doc: hpf and softperipherals comparison (#23226)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/coprocessors/index.rst

   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.

> You can use the VPR coprocessor (Fast Lightweight Peripheral Processor - FLPR) as follows:

Reviewer: Keep the mention of the hardware platforms this section is designated for. VPR is also on nRF54H20, so it's not only nRF54L.

> The VPR coprocessor (Fast Lightweight Peripheral Processor - FLPR) can be utilized as follows:

Reviewer 2: ```suggestion
You can use the VPR coprocessor (Fast Lightweight Peripheral Processor - FLPR) as follows:
```
